searchtool/server/views/layout.hbs

```diff
@@ -193,6 +193,14 @@
                   </div>
                 </div>
                 {{#if accessOK}}
+                
+                <!-- value="{{#if term}}{{term}}{{/if}}" -->
+
+                <div>
+                  <input type="text" name="dn_dw_dn_gau_cd"  aria-label="Search Art Unit Number" title="Search Art Unit Number" class="form-control" placeholder="Search Art Unit Number" value="{{#if artUnit}} {{artUnit}}{{/if}}">
```

Comment:
Can we rename this `art-unit` or something that makes more semantic sense to our future selves?